src/vips/cdefs.lua

```diff
@@ -37,51 +37,51 @@ ffi.cdef [[
 
     int vips_enum_from_nick (const char* domain,
         GType gtype, const char* str);
-    const char *vips_enum_nick (GType gtype, int value);
+    const char* vips_enum_nick (GType gtype, int value);
```

Comment:
I usually prefer C style, but I agree, perhaps C++ style is better.

Reply:
Yeah, I think I was trying to unify the file at that time. I switched to C-style decls with commit https://github.com/libvips/lua-vips/pull/29/commits/c342d9497d1789667df29a737ec30a038df5e3d3.